community.html

```diff
@@ -106,12 +106,13 @@
     <section class="feature">
         <span class="image2"><img src="images/sensorhub.png" alt="" /></span>
         <div class="content2">
-            <h3>SensorHub</h3>
+            <h3>OpenSensorHub</h3>
             <p>
-                SensorHub software allows one to easily build interoperable and evolutive sensor
-                networks with advanced processing capabilities and based on open-standards for
-                all data exchanges.
-                <br/><a href="https://github.com/sensiasoft/sensorhub-core">Source Code</a>
+                OpenSensorHub (OSH) is an open standards, open source software platform to support 
+                virtually any sensor, actuator, or process through the deployment of distributed, 
+                interactive sensor hubs. OSH supports both server and client development.
+                <br/><a href="https://opensensorhub.org">Web Page</a>
```

Comment:
Looks like other listing use the term "Homepage". I'll probably change this to that, 👍